cruise.umple/src/Main_Code.ump

```diff
@@ -292,6 +318,7 @@ class UmpleConsoleMain
     
     String languages = String.join(",", UmpleModel.validLanguages );
     optparser.acceptsAll(Arrays.asList("version", "v"), "Print out the current Umple version number");
+      optparser.acceptsAll(Arrays.asList("performance", "p"), "Indicate time to generate files");
```

Comment:
change this `Indicate time to generate files" with the following one:
`Indicate time taken to parse and generate code`